dss/collections/owner_lookup.py

```diff
@@ -20,10 +20,10 @@ def put_collection(owner: str, collection_fqid: str, permission_level: str = 'ow
 
 
 def get_collection(owner: str, collection_fqid: str):
-    return dynamodb.get_item(table=collection_db_table,
-                             hash_key=owner,
-                             sort_key=collection_fqid,
-                             return_key='sort_key')
+    collection = dynamodb.get_item(table=collection_db_table,
+                                   hash_key=owner,
+                                   sort_key=collection_fqid)
+    return collection.get('sort_key')
```

Comment:
This method seems to return a collection's sort key, not the collection itself; a docstring would be useful here

Reply:
`s/get_collection/get_collection_item/`, perhaps?

Reply:
this code functionality does not change from the previous method of `get_item` a return_key could be used to cherry pick a value out of the item object, with the proposed changes it has to be specified what is wanted out of the item object from dynamodb...
Not sure about why its named `get_collection`; nor do i understand why its passing a known value of the sort_key as a parameter, then returning that same parameter value from the response...
let me do some inspecting around here....

Reply:
it looks like this is only being used within integration tests.

Reply:
Does it make sense to move this to `tests/__init__.py` or somewhere similar?

Reply:
it might, but the functionality is actually interacting with the aws backend..... @DailyDreaming you have an option on this?

Reply:
It's used here: https://github.com/DataBiosphere/data-store/blob/master/scripts/update_collection_db.py#L105  Not just in testing.  It's necessary to populate the collection table from a bucket.